database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java

```diff
@@ -51,7 +51,7 @@
  *            contained in the children of the given Firebase location
  */
 public abstract class FirebaseListAdapter<T> extends BaseAdapter {
-    private static final String TAG = FirebaseListAdapter.class.getSimpleName();
+    private static final String TAG = "FirebaseListAdapter";
```

Comment:
@puf This isn't so much needed in `FirebaseListAdapter`, but this is a massive no-no in the [`FirebaseRecyclerAdapter`](https://github.com/firebase/FirebaseUI-Android/pull/488/files/17a136d524ad4840f49bea54ba740016b7245b19#diff-724a30131c3284a3a9d723d6bb38561dR76) where the name is 23 characters long. I made this tag in one of my first PRs and I really regret it: if someone were to create a tag using `getSimpleName()` in `FirebaseIndexRecyclerAdapter`, `Log.anything` will [throw an exception](http://stackoverflow.com/questions/28168622/the-logging-tag-can-be-at-most-23-characters) because the tag name exceeds 24 characters. You should _**NEVER**_ use dynamic tag names for that reason.

@puf, since this is higher priority than #488, I've create #525.